server.js

```diff
@@ -76,16 +75,11 @@ function buildMentionSentence(reviewers) {
   );
 }
 
-function defaultMessageGenerator(reviewers, pullRequester) {
-  return util.format(
-    '%s, thanks for your PR! ' +
-    'By analyzing the blame information on this pull request' +
-     ', we identified %s to be%s potential reviewer%s',
-     pullRequester,
-     buildMentionSentence(reviewers),
-     reviewers.length > 1 ? '' : ' a',
-     reviewers.length > 1 ? 's' : ''
-  );
+function messageGenerator(message, reviewers, pullRequester) {
+  // replace all @reviwers. message may have more than one '@reviwers'.
+  var withReviwers = message.split('@reviwers').join(buildMentionSentence(reviewers));
+  // replace all @pullRequester, and return
+  return withReviwers.split('@pullRequester').join(pullRequester);
```

Comment:
This is a very interesting way to do replace all in js.

Can you instead do

withReviewers.replace(/@pullRequester/g, pullRequester)

This will be easier for people to understand what this does